alembic/runtime/migration.py

```diff
@@ -697,13 +698,14 @@ def _update_version(self, from_, to_):
                 == literal_column("'%s'" % from_)
             )
         )
-        if not self.context.as_sql and ret.rowcount != 1:
-            raise util.CommandError(
-                "Online migration expected to match one "
-                "row when updating '%s' to '%s' in '%s'; "
-                "%d found"
-                % (from_, to_, self.context.version_table, ret.rowcount)
-            )
+        if not self.context.as_sql and ret.supports_sane_rowcount():
```

Comment:
same

Reply:
addressed in https://github.com/sqlalchemy/alembic/commit/a0d45ed0f5de582314faae2210eeec881670488e